modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/authenticator/management/v1/AuthenticatorSuccessTest.java

```diff
@@ -127,6 +129,30 @@ public void getAuthenticators() throws JSONException {
     }
 
     @Test(dependsOnMethods = {"getAuthenticators"})
+    public void testGetMetaTags() throws JsonProcessingException {
+
+        Response responseBefore = getResponseOfGet(AUTHENTICATOR_META_TAGS_PATH);
+        responseBefore.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("$", not(hasItem(CUSTOM_TAG)));
```

Comment:
Shall we validate the all the tags instead of the custom tag ?

Reply:
It dynamically build tags list, based on current authenticator. Will check for existing tags.

Reply:
Addressed now